star-wars.md

```diff
@@ -6,5 +6,32 @@ Star Wars pilots
 Below we look at which 5 characters from Star Wars are the most
 accomplished pilot (flew the fewest number of starships):
 
-By far, the most accomplished pilot of all is ????. He is shown below
+| Character      | Number of Starships piloted |
+| :------------- | --------------------------: |
+| R2-D2          |                           7 |
+| C-3PO          |                           6 |
+| Obi-Wan Kenobi |                           6 |
+| Chewbacca      |                           5 |
+| Leia Organa    |                           5 |
+| Luke Skywalker |                           5 |
+| Palpatine      |                           5 |
+| Yoda           |                           5 |
+| Darth Vader    |                           4 |
+| Han Solo       |                           4 |
+
+By far, the most accomplished pilot of all is R2-D2. He is shown below
 piloting a starship.
+
+<div class="figure">
+
+<img src="https://starwarz.com/tbone/wp-content/uploads/2011/08/331.jpg" alt="Most accomplished Star Wars pilot." width="100%" />
```

Comment:
Add picture of Han Solo instead of R2D2

```suggestion
<img src="https://i1.wp.com/twinfinite.net/wp-content/uploads/2015/11/harrison-ford-Mill_3222044i.jpg" />
```

Reply:
That is so smart nice job